tests/models/whisper/test_feature_extraction_whisper.py

```diff
@@ -231,6 +226,24 @@ def test_integration(self):
         self.assertEqual(input_features.shape, (1, 80, 3000))
         self.assertTrue(torch.allclose(input_features[0, 0, :30], EXPECTED_INPUT_FEATURES, atol=1e-4))
 
+    def test_numpy_integration(self):
+        # fmt: off
+        EXPECTED_INPUT_FEATURES = np.array(
+            [
+                0.1193, -0.0946, -0.1098, -0.0196, 0.0225, -0.0690, -0.1736, 0.0951,
+                0.0971, -0.0817, -0.0702, 0.0162, 0.0260, 0.0017, -0.0192, -0.1678,
+                0.0709, -0.1867, -0.0655, -0.0274, -0.0234, -0.1884, -0.0516, -0.0554,
+                -0.0274, -0.1425, -0.1423, 0.0837, 0.0377, -0.0854
+            ]
+        )
+        # fmt: on
+
+        input_speech = self._load_datasamples(1)
+        feature_extractor = WhisperFeatureExtractor()
+        input_features = feature_extractor(input_speech, return_tensors="np").input_features
+        self.assertEqual(input_features.shape, (1, 80, 3000))
+        self.assertTrue(np.allclose(input_features[0, 0, :30], EXPECTED_INPUT_FEATURES, atol=1e-4))
```

Comment:
This doesn't actually test numpy integration right ? Torch is installed by default in the test setup ?

This is how I've done it with ASTransformers: https://github.com/huggingface/transformers/blob/cd9f9d63f11890ca34263ec29a5f60ee68e6270a/tests/models/audio_spectrogram_transformer/test_feature_extraction_audio_spectrogram_transformer.py#L204-L222

Reply:
Good shout - resolved in https://github.com/huggingface/transformers/pull/26119/commits/e890cffb248243ad5b1ccffbe43abf237fdbb79c